vue/src/components/UploadForm.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <b-card>
+    <b-card-body>
+      <b-alert variant="danger" v-model="isFailed">{{ errorMessage }}</b-alert>
+      <b-alert variant="success" v-model="isSuccess">Successfully uploaded {{ file.name }}</b-alert>
+      <b-form v-if="!isSuccess" v-on:submit.prevent="submit">
+        <b-form-group>
+          <b-form-file v-model="file"
+                       accept="audio/*"
```

Comment:
i'm getting a little confused about all these frontends building in file extension conditions, since pretty much everyone says this isn't very secure?

Reply:
also, i didn't see WAVE in the IANA document, did it work for you?

Reply:
This is for UX, not security. It makes the file selector browser smarter.

Reply:
Yeah wav uploads worked for me